easybuild/easyconfigs/s/SortMeRNA/SortMeRNA-2.1-foss-2016a.eb

```diff
@@ -0,0 +1,37 @@
+##
+# This file is an EasyBuild reciPY as per https://github.com/hpcugent/easybuild
+#
+# Copyright:: Copyright 2012-2014 Uni.Lu/LCSB, NTUA
+# Authors::   Cedric Laczny <cedric.laczny@uni.lu>, Fotis Georgatos <fotis@cern.ch>
+# License::   MIT/GPL
+# $Id$
+#
+# This work implements a part of the HPCBIOS project and is a component of the policy:
+# http://hpcbios.readthedocs.org/en/latest/HPCBIOS_2012-94.html
+##
+easyblock = 'ConfigureMake'
+
+name = 'SortMeRNA'
+version = '2.1'
+
+homepage = 'http://bioinfo.lifl.fr/RNA/sortmerna/' 
+description = """
+SortMeRNA is a biological sequence analysis tool for filtering, mapping and OTU-picking NGS reads.
+""" 
+
+# Patch for double includes of header files in Makefile.in
+patches=['SortMeRNA_Makefile.in.patch']
```

Comment:
@RvDijk please move this down below `sources`, and include the version in the patch filename using `%(version)s` (since the patch may easily become incompatible with future versions